aad_fastapi/aad_options.py

```diff
@@ -27,18 +27,18 @@ class AzureAdSettings(BaseSettings):
 
     """
 
-    client_id: str = Field(None, description="Client id", env="CLIENT_ID")
-    client_secret: str = Field(None, description="Client Secret", env="CLIENT_SECRET")
-    authority: str = Field(None, description="login authority", env="AUTHORITY")
-    domain: str = Field(None, description="Domain name", env="DOMAIN")
-    tenant_id: str = Field(None, description="Tenant Id", env="TENANT_ID")
-    scopes: str = Field(None, description="Scopes", env="SCOPES")
-    vault_name: str = Field(None, description="Global Vault Url", env="VAULT_NAME")
+    client_id: str = Field("", description="Client id", env="CLIENT_ID")
```

Comment:
Hey @dorlugasigal, I'm not sure about that, replacing `None` with `""`, but at least it's working
Maybe `Optional` would be better (but It did not work for me, I guess I made a mistake somewhere)

Reply:
Using `Optional[str]` resolves this.